cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -34,7 +39,7 @@ def test_no_dvd_ruins_refresh(provider, testing_vm_without_dvd):
 
 @pytest.mark.manual
 @pytest.mark.tier(2)
-def test_template_info_scvmm2016():
+def test_template_info_scvmm():
```

Comment:
optional:
If you are trying to give good `id` then john best hack is inherit `provider` :)